sql/testdata/builtin_function

```diff
@@ -573,4 +575,4 @@ SELECT translate('12345', 'abc', 'ax')
 query T
 SELECT translate('a‰ÒÁ', 'aÒ', '∏p')
 ----
-∏‰pÁ
\ No newline at end of file
+∏‰pÁ
```

Comment:
The nonsense characters are intentional - this test is verifying that `translate()` works with non-ascii characters. Your editor automatically added a trailing newline but the last person to touch this file didn't.